dotCMS/src/main/java/com/dotcms/graphql/exception/FieldGenerationException.java

```diff
@@ -0,0 +1,18 @@
+package com.dotcms.graphql.exception;
+
+import com.dotmarketing.exception.DotRuntimeException;
+
+public class FieldGenerationException extends DotRuntimeException {
+
+    public FieldGenerationException(String message) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'message' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)